backend/schemas/dashboardTile.js

```diff
@@ -5,7 +5,11 @@ var tiles = mongoose.Schema({
     "displayTitle": String,
     "description": String,
     "_poster": String,
-    "_link": String,
+    "_tileLinkType": String,
```

Comment:
Why has this changed name - surely this would break anyone who has linked this before hand

Reply:
I changed it because now link can refer to course links or external links, I wanted it to be more precise than just link. I can change _courseExternalLink to _link so that any existing ones won't break?

Reply:
Changes made.